### PR TITLE
MINOR: Upgrade jetty to 9.4.30.v20200611 and jersey to 2.31

### PR DESCRIPTION
**Pull request by kkonstantine**
Align versions with AK.

### CODE REVIEW

**Comment by kkonstantine** [comment]
We are hitting an issue with an incompatible change in jetty described here: 
https://github.com/eclipse/jetty.project/issues/4693
This change (the renaming of `closeOutput` to `completeOutput` is reverted in more recent versions). 
Oldest version that has a fix is: `jetty-9.4.28.v20200408` by looking at https://github.com/eclipse/jetty.project/releases
Probably best to upgrade to this version or newer instead of `9.4.27.v20200227`

Wdyt @ijuma ?

**Comment by ijuma** [comment]
@kkonstantine The jersey upgrade was supposed to have fixed that since 9.4.27 is what they actually test it according to the submitter.

**Comment by kkonstantine** [comment]
I see. I didn't check `jersey`. But we are hitting an issue in SR where the jetty method `Response#closeOutput` is used [directly](https://github.com/confluentinc/schema-registry/blob/master/core/src/main/java/io/confluent/kafka/schemaregistry/rest/JsonErrorHandler.java#L54) from jetty.

File: pom.xml [hunk 1]
```diff
@@ -50,8 +50,8 @@
         <activation.version>1.1.1</activation.version>
         <apache.httpclient.version>4.5.3</apache.httpclient.version>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
-        <jersey.version>2.30</jersey.version>
-        <jetty.version>9.4.24.v20191120</jetty.version>
+        <jersey.version>2.31</jersey.version>
+        <jetty.version>9.4.27.v20200227</jetty.version>
```
Comment (kkonstantine) [inline pom.xml:54]:
```suggestion
        <jetty.version>9.4.30.v20200611</jetty.version>
```

**Comment by kkonstantine** [comment]
The build is now passing with jetty version `9.4.30.v20200611`.

@ijuma if we are good to go here, I'll merge and submit a PR in AK to align the versions there as well.

**Comment by kkonstantine** [comment]
The equivalent PR in AK has been merged: https://github.com/apache/kafka/pull/8893 
Aligning the versions here too. Thanks @ijuma @ewencp

**Pull request merged**